src/NuGet.Clients/NuGet.CommandLine/NuGetResources.resx

```diff
@@ -6177,4 +6177,7 @@ Oluşturma sırasında NuGet'in paketleri indirmesini önlemek için, Visual Stu
     <value>The action '{0}' is not recognized.</value>
     <comment>0 - action</comment>
   </data>
+  <data name="ListCommandDeprecatedMessage" xml:space="preserve">
+    <value>list command is deprecated. Use search command instead.</value>
```

Comment:
I know this won't get localized, so it's not a problem, but whne using keywords, make sure you add a comment or pass them in as parameters.

Reply:
Also consider using quotes around the keywords